CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 88 Release Notes
 
+###  Mysql 8.4
+
+Cromwell now supports MySQL 8.4 and 8.4-compatible databases. Older versions of MySQL are no longer officially supported or tested, but they are not explicitly forbidden from use.
```

Comment:
Do we have a specific reason to also drop 8.0?

Its extended support date (which is what CloudSQL uses) is [through 2026](https://endoflife.date/mysql).

Reply:
oh sure I can add that back! I thought since we'd upgraded to 8.4 that was the only necessary version, but after re-reading the ticket I can see I was too trigger happy